src/live/src/user-to-user/live-service-facade.js

```diff
@@ -0,0 +1,58 @@
+import PubNub from 'pubnub';
+
+import Client from '../../../client';
+import { User } from '../../../entity';
+import { PubNubListener } from './pubnub-listener';
+import { getLiveService } from './live-service';
+
+
+function _getLiveService() {
+  const client = Client.sharedInstance();
+  return getLiveService(client);
+}
+
+function register() {
+  const liveService = _getLiveService();
+  const activeUser = User.getActiveUser();
+
+  return liveService.registerUser(activeUser)
+    .then((pubnubConfig) => {
+      const pubnubClient = new PubNub(pubnubConfig);
+      const listener = new PubNubListener();
+      liveService.initialize(pubnubClient, listener);
+    });
+}
+
+function unregister() {
+  const liveService = _getLiveService();
+  return liveService.unregisterUser()
+    .then((resp) => {
+      liveService.uninitialize();
+      return resp;
+    });
+}
+
+function onConnectionStatusUpdates(func) {
+  _getLiveService().onConnectionStatusUpdates(func);
+}
+
+function offConnectionStatusUpdates(func) {
+  _getLiveService().offConnectionStatusUpdates(func);
+}
+
+function unsubscribeFromAll() {
+  _getLiveService().unsubscribeFromAll();
+}
+
+function isInitialized() {
```

Comment:
To me it seems like we are requiring the end user to understand more about PubNub and how it works. They first have to register a user but why do I need to know if something is initialized? What is initialized? Did I register my user? Doesn't that initialize me?

These are the kind of questions I would have if I was a developer. Our goal is to make it as simple as possible so the end user doesn't need to understand the internals.